### PR TITLE
Add installation hints and note for beamer support

### PR DESCRIPTION
**Pull request by gannimo**
The required packages and installation instructions are incomplete.
* Adding notes on what packages are required on Debian (and likely Ubuntu).
* Adding notes to update the template for LaTeX beamer users

### CODE REVIEW

**Comment by Hakuyume** [comment]
Thank you for your PR!

The rendered result seems broken. `$ apt install` should be block quoted (currently, it is inline quoted). Could you fix it?
https://github.com/Hakuyume/pandoc-filter-graphviz/blob/5b8364cffc64f520dfc22c64f8976323952df8f4/README.md

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ This filter provides Graphviz code block support.
 - graphviz
 - dot2tex (pdf/LaTeX support)
 
+On Debian, you can install the required packages as follows:
+    ```
+    $ sudo apt install ghc libghc-utf8-string-dev libghc-sha-dev libghc-pandoc-dev
```
Comment (Hakuyume) [inline README.md]: Shouldn't `graphviz` and `dot2tex` be included here?

Comment (gannimo) [reply README.md]: They are pulled in as dependencies, I only added the minimum amount of packages required. The dependency management handles the rest.

**Review by gannimo — COMMENTED** [review]
> Thank you for your PR!
> 
> The rendered result seems broken. `$ apt install` should be block quoted (currently, it is inline quoted). Could you fix it?
> https://github.com/Hakuyume/pandoc-filter-graphviz/blob/5b8364cffc64f520dfc22c64f8976323952df8f4/README.md

To get a block quote, the leading spaces will have to be removed.
Do you want me to create a new pull request?

File: README.md (hunk 1 above)
Comment (Hakuyume) [reply README.md]: I meant we should also include `pandoc`, `graphviz`, and `dot2tex` in the instruction. I tried your command but it didn't pull these three packages.

Comment (gannimo) [reply README.md]: OK, I've added these packages to the overall list. Thanks!

**Comment by Hakuyume** [comment]
LGTM. Thank you!

**Pull request merged**